runbotics/runbotics-orchestrator-ui/src/views/process/ProcessBuildView/Modeler/ConfigureActionPanel/widgets/ElementAwareAutocompleteWidget.tsx

```diff
@@ -145,13 +157,42 @@ const ElementAwareAutocompleteWidget: FC<WidgetProps> = (props) => {
                         element.businessObject.actionId === 'variables.assignList',
                 );
 
-            const variables = assignVariablesElements
+            const globalVariables = assignGlobalVariablesElements
+                .map((assignGlobalVariablesElement) => {
+                    const inputOutput: CamundaInputOutputElement = assignGlobalVariablesElement.businessObject
+                        ?.extensionElements?.values[0] as CamundaInputOutputElement;
+                    if (inputOutput) {
```

Comment:
make it a guard `if`

invert if condition and return undefined in it then the rest does not need to be in if scope